### ISSUE TITLE
Chunk size in parfor_outer

### ISSUE THREAD

**Issue by Norbo11** [HPCE/hpce-2018-cw2#78]
Is the advice given in #62 still valid? We should let TBB auto-partition our space?

**Comment by jjd06** [comment]
Yes -- feel free to leave it as-is (or not if you find you can do better manually!).
